editor.planx.uk/src/@planx/components/shared/Preview/QuestionHeader.tsx

```diff
@@ -28,13 +28,13 @@ const Description = styled(Box)(({ theme }) => ({
   },
 }));
 
-const StyledIconButton = styled(IconButton)(() => ({
+export const StyledIconButton = styled(IconButton)(() => ({
```

Comment:
nit: This might be best moving to the `editor.planx.uk/src/ui`folder (and having a `size` props?)